localization/english/wc_government_l_english.yml

```
@@ -0,0 +1,12 @@
+﻿l_english:
+ necro_government:0 "Necro"
+ necro_government_adjective:0 "Necro"
+ necro_government_realm:0 "Necro Realm"
+ necro_government_desc:0 "\n$game_concept_necro_government_desc$\n\n#F A realm ruled by the dead. Violent by nature and incapable of reproduction, they have to kill the living so they will join them in the afterlife.#!"
```

Changed the latter part.
```suggestion
 necro_government_desc:0 "\n$game_concept_necro_government_desc$\n\n#F A realm ruled by the dead. Violent by nature and incapable of reproduction, they have to kill and resurrect the living in order to replenish their numbers.#!"
```